modules/api/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@luma.gl/api",
+  "version": "8.5.1",
+  "description": "luma.gl API",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/visgl/luma.gl"
+  },
+  "keywords": [
+    "webgl",
+    "visualization",
+    "animation",
+    "3d"
+  ],
+  "types": "src/index.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "files": [
+    "src",
+    "dist",
+    "README.md"
+  ],
+  "sideEffects": false,
+  "browser": {
```

Comment:
Remove? Given gl is not a dependency it should never be referenced.

Reply:
Yes. Will update in next PR